src/mocks/products.JSON

```
@@ -0,0 +1,58 @@
+[
+  {
+    "id": 1,
+    "name": "Syltherine",
+    "description": "description.product1",
+    "price": 2500000,
+    "image": "/catalogue/Product1.webp"
+  },
+  {
+    "id": 2,
+    "name": "Leviosa",
+    "description": "description.product2",
+    "price": 2500000,
+    "image": "/catalogue/Product1.webp"
+  },
+  {
+    "id": 3,
+    "name": "Lolito",
+    "description": "description.product3",
```

Just to give u a suggestion on how this may come on the real world:

```suggestion
    "description": {
         'es': 'Spanish description of the product',
         'en': 'English description of the product',
         ...
    },
```

And then you use the current language id (es, en) to render according to it.

Another can be sending the language id key on the product /GET request on the hypotetical API to receive the copies woth the desired language